io.github.fastrizwaan.WineZGUI.yml

```diff
@@ -0,0 +1,151 @@
+id: io.github.fastrizwaan.WineZGUI
+branch: '23.08'
+version: '0.97.12'
```

Comment:
this is an invalid manifest property. flatpak-builder likely emitted a warning when you built this locally.